ComputerInterface/ComputerInterfaceCallbacks.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+using UnityEngine;
+
+using Photon.Pun;
+using Photon.Realtime;
+
+namespace ComputerInterface
+{
+    class ComputerInterfaceCallbacks : MonoBehaviourPunCallbacks

```

Comment:
Are these used anywhere?

Reply:
They were for something that I deleted, I thought I deleted this too